src/components/VariablesDeleteButton.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <p-button v-if="variableIds.length > 0" danger icon="TrashIcon" @click="open" />
+  <ConfirmDeleteModal
+    v-model:showModal="showModal"
+    :name="localization.info.selectedVariables"
+    :label="localization.info.variables"
+    @delete="deleteVariables(variableIds)"
+  />
+</template>
+
+<script lang="ts" setup>
+  import { showToast } from '@prefecthq/prefect-design'
+  import { computed } from 'vue'
+  import ConfirmDeleteModal from '@/components/ConfirmDeleteModal.vue'
+  import { useShowModal, useWorkspaceApi } from '@/compositions'
+  import { localization } from '@/localization'
+  import { toPluralString } from '@/utilities'
+
+  defineProps<{
+    variableIds: string[],
+  }>()
+
+  const emit = defineEmits<{
+    (event: 'delete'): string[],
+  }>()
+
+  const { showModal, open, close } = useShowModal()
+
+  const api = useWorkspaceApi()
+
+  const deleteVariables = async (variableIds: string[]): Promise<void> => {
+    try {
+      const variableDeletePromises = variableIds.map(api.variables.deleteVariable)
+      await Promise.all(variableDeletePromises)
```

Comment:
Since this feature is being actively built. Any chance we can get an api endpoint for this?